Common/cpp/SharedItems/ShareableValue.cpp

```diff
@@ -11,13 +11,11 @@ namespace reanimated {
 
 const char *HIDDEN_HOST_OBJECT_PROP = "__reanimatedHostObjectRef";
 const char *ALREADY_CONVERTED= "__alreadyConverted";
-std::string CALLBACK_ERROR_PREFIX = R"(
-Tried to synchronously call function {)";
-std::string CALLBACK_ERROR_SUFFIX = R"(} from a different thread.
+std::string CALLBACK_ERROR_SUFFIX = R"(
+
 Solution is:
```

Comment:
For me this is kinda awkward wording, I'd remove it altogether.

Reply:
In my opinion It can be helpful, but maybe we should change “Solution is” -> “Possible solutions are“ 
It means that these solutions are just some of many. What do you think?